cypress/e2e/sample_test_spec/SampleTestRequest.cy.ts

```diff
@@ -0,0 +1,42 @@
+import { SampleTestPage } from "pageobject/Sample/SampleTestCreate";
+
+describe("Sample Test", () => {
+  const sampleTestPage = new SampleTestPage();
+  const sampleTestType = "BA/ETA",
+    icmrCategory = "Cat 0",
+    icmrLabel = "Test Icmr Label";
+
+  before(() => {
+    cy.loginByApi("devdistrictadmin", "Coronasafe@123");
+    cy.saveLocalStorage();
+  });
+
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+    cy.clearLocalStorage(/filters--.+/);
+  });
+
+  it("should request a new sample test", () => {
+    sampleTestPage.visitPatientPage();
+    sampleTestPage.visitPatientDashboardPage();
+    sampleTestPage.visitSampleRequestPage();
+
+    // Fill form fields
+    sampleTestPage.selectSampleType(sampleTestType);
+    sampleTestPage.selectIcmrCategory(icmrCategory);
+    sampleTestPage.typeIcmrLabel(icmrLabel);
+
+    // Submit the form
+    sampleTestPage.submitForm();
+
+    // Check for sample request notification and history
+    sampleTestPage.clickOnNotification();
+    sampleTestPage.checkRequestHistory();
+  });
+
+  it("should verify sample request on sample page", () => {
+    sampleTestPage.visitSamplePage();
+    sampleTestPage.searchPatientSample(sampleTestPage.patientName);
```

Comment:
```suggestion
    sampleTestPage.searchPatientSample(sampleTestPage.patientName);
```

- check for the patient card presence, and verify the patient name in the card
- then click on the sample details button, and verify all the inputted details are properly present in the sample details page
- add a cy.intercept to ensure that the API status to ensure the page is fully loaded